server/src/services/library.service.ts

```diff
@@ -330,6 +336,9 @@ export class LibraryService extends BaseService {
   async handleDeleteLibrary(job: JobOf<JobName.LIBRARY_DELETE>): Promise<JobStatus> {
     const libraryId = job.id;
 
+    // Hide all assets in the library before deleting them
```

Comment:
```suggestion
    // Soft delete all assets in the library
```
I personally think this comment isn't even necessary

Reply:
I removed it